easybuild/easyconfigs/h/HDF/HDF-4.2.16-2-GCCcore-12.3.0.eb

```
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'HDF'
+version = '4.2.16-2'
+
+homepage = 'https://www.hdfgroup.org/products/hdf4/'
+description = """
+ HDF (also known as HDF4) is a library and multi-object file format for
+ storing and managing data between machines.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.hdfgroup.org/ftp/%(name)s/releases/%(name)s%(version)s/src/']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = [
+    'a24b18312d421686031c2d66635f7d5abb2fe879f8a182b7e02797b0da8d1f6c',  # %(namelower)s-%(version)s.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+]
+dependencies = [
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('Szip', '2.1.1'),
+    ('zlib', '1.2.13'),
+    ('libtirpc', '1.3.3'),
+]
+
+preconfigopts = "LIBS='-ltirpc' "
+configopts = [
+    '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc"',
+    '--includedir=%(installdir)s/include/%(namelower)s FFLAGS="$FFLAGS -fallow-argument-mismatch"',
+    '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc"',
+    '--includedir=%(installdir)s/include/%(namelower)s',
+    '--enable-shared',
+    '--disable-fortran',
+    '--disable-netcdf',
+]
+
```

```suggestion
local_common_configopts = '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc" '
local_common_configopts += '--includedir=%(installdir)s/include/%(namelower)s '
configopts = [
    local_common_configopts,
    # Cannot build shared libraries and Fortran...
    # https://trac.osgeo.org/gdal/wiki/HDF#IncompatibilitywithNetCDFLibraries
    # netcdf must be disabled to allow HDF to be used by GDAL
    local_common_configopts + "--enable-shared --disable-fortran --disable-netcdf",
]
```

With `configopts` as a list we end up building once per item in the list.